cmd/telemetrygen/internal/traces/config.go

```diff
@@ -26,7 +26,7 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 	fs.IntVar(&c.NumTraces, "traces", 1, "Number of traces to generate in each worker (ignored if duration is provided)")
 	fs.BoolVar(&c.PropagateContext, "marshal", false, "Whether to marshal trace context via HTTP headers")
 	fs.StringVar(&c.ServiceName, "service", "telemetrygen", "Service name to use")
-	fs.StringVar(&c.StatusCode, "status-code", "Unset", "Status code to use for the spans, one of (Unset, Error, Ok) or the equivalent integer (0,1,2)")
+	fs.StringVar(&c.StatusCode, "status-code", "0", "Status code to use for the spans, one of (Unset, Error, Ok) or the equivalent integer (0,1,2)")
```

Comment:
I'm confused by why Unset doesn't work, can you open a separate issue to address this?

Reply:
Ok, I will open a seperate pr.